### PR TITLE
removed extra white spaces and typos

### PR DESCRIPTION
**Pull request by fauzank339**
#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by wermos** [comment]
Hi, it seems that this is your first PR. Since this is your first time, you also have to add yourself to the `.mailmap` file. You can find the instructions for doing so [here](https://github.com/sympy/sympy/wiki/Adding-Someone-to-AUTHORS).

**Comment by fauzank339** [comment]
Hi I am not able to add myself to author, 
I cloned the repository went to the bin folder and found author_update.py file
tried to run that file but nothing happened. Could you help me?

**Comment by fauzank339** [comment]
I also tried running mailmap_check and mailmap_update in my python IDE 
Nothing happened, I need some conceptual clarity as well here, Do we have to run it in git or in terminal or can it also be achieve by running it on ide.

**Comment by oscarbenjamin** [comment]
See the instructions here:
https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file

You need to add your name and email address to the .mailmap file. Then in a terminal run
```console
$ python bin/mailmap_check.py
```

File: doc/src/tutorial/intro.rst [hunk 1]
```diff
@@ -6,33 +6,33 @@ What is Symbolic Computation?
 =============================
 
 Symbolic computation deals with the computation of mathematical objects
-symbolically.  This means that the mathematical objects are represented
+symbolically. This means that the mathematical objects are represented
```
Comment (oscarbenjamin) [inline doc/src/tutorial/intro.rst:9]: I don't think that these kinds of changes are really needed.

**Comment by oscarbenjamin** [comment]
It looks like you have run the wrong file for the authors update. It should be `mailmap_check.py` not `mailmap_update.py`.

File: doc/src/tutorial/intro.rst (hunk 1 above)
Comment (fauzank339) [reply doc/src/tutorial/intro.rst:9]: oh Sorry, I thought those spaces were there by mistake, my next contribution will be a worthy contribution. I am working on an issue, which is an Easy to fix issue. Thanks.

File: doc/src/tutorial/intro.rst [hunk 2]
```diff
@@ -6,33 +6,33 @@ What is Symbolic Computation?
...
    3.0
 
 9 is a perfect square, so we got the exact answer, 3. But suppose we computed
-the square root of a number that isn't a perfect square
+the square root of a number that isn't a perfect square.
 
    >>> math.sqrt(8)
    2.82842712475
 
 Here we got an approximate result. 2.82842712475 is not the exact square root
```
Comment (eagleoflqj) [inline doc/src/tutorial/intro.rst:26]: I reverted the change here. Two complete sentences shouldn't be joined by comma.

File: doc/src/tutorial/intro.rst (hunk 1 above)
Comment (eagleoflqj) [reply doc/src/tutorial/intro.rst:9]: This PR shouldn't stay open forever. Let's either merge or close.
I weakly prefer merge as it does no harm to doc itself (cons: increase repo size, more commits when bisecting).

File: .mailmap [hunk 3]
```diff
@@ -1397,6 +1398,7 @@ dustyrockpyle <dustyrockpyle@gmail.com>
 elvis-sik <e.sikora@grad.ufsc.br>
 erdOne <36414270+erdOne@users.noreply.github.com>
 faizan2700 <syedfaizan824@gmail.com>
+fauzank339 <50906187+fauzank339@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline .mailmap:1401]: Only one name should be added to .mailmap. Which name do you want to be in the AUTHORS file?

File: doc/src/tutorial/intro.rst (hunk 1 above)
Comment (oscarbenjamin) [reply doc/src/tutorial/intro.rst:9]: It doesn't do significant harm but it also doesn't really improve anything. Opinions differ on whether there should be one or two spaces after a full stop. Someone could just as easily come along in future and try to change all of these back. As far as I know this doesn't actually affect the built docs. If we want to have a uniform opinion about this then it should be in the style guide:
https://docs.sympy.org/latest/guides/contributing/documentation-style-guide.html

Comment (eagleoflqj) [reply doc/src/tutorial/intro.rst:9]: Make sense. Then we should probably close it.

**Comment by eagleoflqj** [comment]
Closing. Looking forward to your next contribution!

**Pull request closed without merging**